admin/satellites/geo-dns.md

```diff
@@ -0,0 +1,120 @@
+---
+title: GeoDNS
```

Comment:
How about we call this "global load balancing," or something similar?

If you google GeoDNS, this is the top hit: https://en.wikipedia.org/wiki/GeoDNS

Do these docs only refer to Anycast routing, or can people use this if they are using split-horizon DNS?